doc/specs/#1595 - Suggestions UI/Snippets.md

```diff
@@ -470,16 +528,32 @@ their own workflows.
   example, see
   [`clone_all_repos_in_org.yaml`](https://github.com/warpdotdev/workflows/blob/main/specs/git/clone_all_repos_in_org.yaml).
   We may want to straight up just seamlessly support that syntax as well.
-  Converting them to WT-compatible json is fairly trivial [[1](#footnote-1)].
-  Furthermore, the commands are all licensed under Apache 2.0, which means they
-  can be easily consumed by other OSS projects and shared with other developers.
-  This leads us to the next future consideration:
+  * Converting them to WT-compatible json is fairly trivial [[2](#footnote-2)].
+    * We may want to consider supporting YAML like this for `wt import`, ala [#10083]
+    * Similarly, we could import the YAML in the settings UI in a fashion similar
+      to how we import color schemes:

```

Comment:
We can/should probably mention the idea of importing .vscode/tasks.json here too. It'd be particularly cool if we could do that automatically since VS Code is also a first-party product.

Reply:
I put that in the future considerations already 😉